translations/de.json

```
@@ -0,0 +1,226 @@
+{  
+    "translation": {  
+        "🔵 Average": "🔵 Durchschnittlich",  
+        "🟣 Flea Banned": "🟣 nicht auf dem Flohmarkt verfügbar",  
+        "🟢 Great": "🟢 Gut",  
+        "⭐ Legendary ⭐": "⭐ Legendär ⭐",  
+        "🔴 Poor": "🔴 Schlecht",  
+        "🛒 {{trader.name}} restock in {{numMinutes}} minutes 🛒": "🛒 {{trader.name}} neue Vorräte in {{numMinutes}} Minuten 🛒",  
+        "❌ ERROR ❌": "❌ Fehler ❌",  
+        "{{thingName}} set to {{level}}.": "{{thingName}} auf {{level}} gesetzt.",  
+        "A Dmg": "A Dmg",  
+        "About": "Über",  
+        "additional results not shown.": "weitere Resultate nicht angezeigt.",  
+        "All hideout stations set to {{level}}.": "Alle Unterschlupf Ausbaumöglichkeiten auf {{level}} gesetzt.",  
+        "All traders set to {{level}}.": "Alle Händler auf {{level}} gesetzt.",  
+        "all traders": "alle Händler",  
+        "Ammo Table": "Munitionstabelle",  
+        "An error has occurred": "Es ist ein Fehler aufgetreten",  
+        "Available Commands": "Verfügbare Befehle",  
+        "Barter": "Händler",  
+        "Bosses": "Bosse",  
+        "Bugs? Missing features? Questions? Chat with us on Discord!": "Fehler gefunden? Fehlende Funktion? Fragen? Sprich mit uns auf Discord!",  
+        "Chance": "Chance",  
+        "Changes provided by": "Änderungen zur Verfügung gestellt von",  
+        "\[Click here\]({{url}}) for full notes.": "\[Klick hier\]({{url}}) für alle Hinweise.",  
```

```suggestion
        "[Click here]({{url}}) for full notes.": "[Klick hier]({{url}}) für alle Hinweise.",  
```
Should be no escapes

---

Fixed in commit [c880c4b](https://github.com/the-hideout/stash/pull/125/commits/c880c4b5f3f62fc19379ca20d84a53b87adb653a)